public/controllers/settings.js

```diff
@@ -291,43 +288,30 @@ let app = require('ui/modules').get('app/wazuh', []).controller('settingsControl
     };
 
     // Check manager connectivity
-    $scope.checkManager = (item) => {
-        let index = $scope.apiEntries.indexOf(item);
+    $scope.checkManager = item => {
+        const index = $scope.apiEntries.indexOf(item);
         
-        let tmpData = {
-            'user':         $scope.apiEntries[index]._source.api_user,
-            'password':     $scope.apiEntries[index]._source.api_password,
-            'url':          $scope.apiEntries[index]._source.url,
-            'port':         $scope.apiEntries[index]._source.api_port,
-            'cluster_info': {},
-            'insecure':     'true',
-            'id':           $scope.apiEntries[index]._id
+        const tmpData = {
+            user:         $scope.apiEntries[index]._source.api_user,
+            password:     $scope.apiEntries[index]._source.api_password,
+            url:          $scope.apiEntries[index]._source.url,
+            port:         $scope.apiEntries[index]._source.api_port,
+            cluster_info: {},
+            insecure:     'true',
+            id:           $scope.apiEntries[index]._id,
+            cluster_info: null
```

Comment:
cluster_info was previously defined. Keep just one, isn't it?

Reply:
That line of code initializes `tmpData` in a const object, after that we give a value to `cluster_info`, before this PR it was cluster_info: {}, but is indiferent cos we assign a direct value after that.